src/AddItems.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+import { capitalize } from './lib/utility';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [userToken, setUserToken] = useState('randomTokenOne');
+  const [itemFreq, setItemFreq] = useState(7);
+  const [formValidationVisual, setFormValidationVisual] = useState(false);
+
+  const updateGroceryItem = (event) => {
+    setGroceryItem(capitalize(event.target.value));
+    setFormValidationVisual(true);
+  };
+
+  const submitGroceryItem = (event) => {
+    event.preventDefault();
+    const formData = {
+      itemName: groceryItem,
+      frequency: itemFreq,
+      lastPurchaseDate: null,
+    };
+    db.collection(userToken).add({ formData });
+    setGroceryItem('');
+    setItemFreq(7);
+    setFormValidationVisual(false);
+  };
+
+  const radioBtnHandler = (event) => {
+    setItemFreq(+event.target.value);
```

Comment:
clever...I would not have though of just adding the `+`.

Reply:
I'm not sure it's working, though:  I just created this item:

![image](https://user-images.githubusercontent.com/10378331/114869207-57f4b700-9dbc-11eb-8ab8-8f3a6bcb0720.png)

I wonder if it is how you are passing in this function to the `onChange`...see my comment there.

Reply:
Good catch! Totally didn't realize I hadn't passed the event argument there so totally makes sense as to why it wasn't working.